.github/release.yml

```diff
@@ -0,0 +1,18 @@
+# .github/release.yml
+
+changelog:
+  exclude:
+    authors:
+      - dependabot
+      - token-generator-for-cchq
+  categories:
+    - title: Product Updates
+      labels:
+        - product/all-users-all-environments
```

Comment:
Should this also include `product/prod-india-all-users`?

Reply:
Added! thanks for pointing that out

> Should this also include `product/prod-india-all-users`?